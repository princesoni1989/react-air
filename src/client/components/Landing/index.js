import React from 'react';
import {NavLink} from "react-router-dom";

export default function Landing() {
  return (
    <div className='container'>
      <div className='welcome'>
        <div className='seperator-line'>
          <h1 className='heading'>
            Welcome To React Boiler Plate
          </h1>
          <NavLink to='/login' className='form-link'>Login</NavLink>
          <NavLink to='/signup' className='form-link'>Sign Up</NavLink>
        </div>
      </div>
    </div>
  );
}
