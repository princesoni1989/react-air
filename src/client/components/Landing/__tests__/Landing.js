/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';

import Landing from '..';

describe(Landing.name, () => {
  it('should render correctly', () => {
    expect(shallow(<Landing />)).toMatchSnapshot();
  });
});
