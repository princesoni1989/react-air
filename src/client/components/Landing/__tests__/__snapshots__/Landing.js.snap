// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Landing />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="welcome"
      >
        <div
          className="seperator-line"
        >
          <h1
            className="heading"
          >
            Welcome To React Boiler Plate
          </h1>
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="form-link"
            to="/login"
          >
            Login
          </NavLink>
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="form-link"
            to="/signup"
          >
            Sign Up
          </NavLink>
        </div>
      </div>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="seperator-line"
        >
          <h1
            className="heading"
          >
            Welcome To React Boiler Plate
          </h1>
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="form-link"
            to="/login"
          >
            Login
          </NavLink>
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="form-link"
            to="/signup"
          >
            Sign Up
          </NavLink>
        </div>,
        "className": "welcome",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="heading"
            >
              Welcome To React Boiler Plate
            </h1>,
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="form-link"
              to="/login"
            >
              Login
            </NavLink>,
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="form-link"
              to="/signup"
            >
              Sign Up
            </NavLink>,
          ],
          "className": "seperator-line",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome To React Boiler Plate",
              "className": "heading",
            },
            "ref": null,
            "rendered": "Welcome To React Boiler Plate",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": "Login",
              "className": "form-link",
              "to": "/login",
            },
            "ref": null,
            "rendered": "Login",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": "Sign Up",
              "className": "form-link",
              "to": "/signup",
            },
            "ref": null,
            "rendered": "Sign Up",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="welcome"
        >
          <div
            className="seperator-line"
          >
            <h1
              className="heading"
            >
              Welcome To React Boiler Plate
            </h1>
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="form-link"
              to="/login"
            >
              Login
            </NavLink>
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="form-link"
              to="/signup"
            >
              Sign Up
            </NavLink>
          </div>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="seperator-line"
          >
            <h1
              className="heading"
            >
              Welcome To React Boiler Plate
            </h1>
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="form-link"
              to="/login"
            >
              Login
            </NavLink>
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="form-link"
              to="/signup"
            >
              Sign Up
            </NavLink>
          </div>,
          "className": "welcome",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="heading"
              >
                Welcome To React Boiler Plate
              </h1>,
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                className="form-link"
                to="/login"
              >
                Login
              </NavLink>,
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                className="form-link"
                to="/signup"
              >
                Sign Up
              </NavLink>,
            ],
            "className": "seperator-line",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Welcome To React Boiler Plate",
                "className": "heading",
              },
              "ref": null,
              "rendered": "Welcome To React Boiler Plate",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Login",
                "className": "form-link",
                "to": "/login",
              },
              "ref": null,
              "rendered": "Login",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Sign Up",
                "className": "form-link",
                "to": "/signup",
              },
              "ref": null,
              "rendered": "Sign Up",
              "type": [Function],
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
